src/Transition/CreateNewIDFUsingRulesV9_3_0.f90

```diff
@@ -1845,4 +1875,139 @@ SUBROUTINE SortUnique(StrArray, Size, Order)
     END DO
   END DO
 
-END SUBROUTINE SortUnique
\ No newline at end of file
+END SUBROUTINE SortUnique
+
+SUBROUTINE UpdateEMSFunctionName(InOutArg, OldName, NewName, NoDiffArg)
+  USE InputProcessor, ONLY: SameString
+  CHARACTER(len=*), INTENT(INOUT) :: InOutArg
+  CHARACTER(len=100) ::  NewInOutArg
+  CHARACTER(len=*), INTENT(IN) :: OldName
+  CHARACTER(len=*), INTENT(IN) :: NewName
+  LOGICAL, INTENT(INOUT) :: NoDiffArg
+
+  PARAMETER(N=20)
+  CHARACTER*20 ARRAY(N)
+  INTEGER ICOUNT, IBEGIN(N),ITERM(N),ILEN, J, LenToken, IDXStart, IDXEnd
+
+  Call DELIM(InOutArg, ARRAY, N, ICOUNT, IBEGIN, ITERM, ILEN, ' ')
+
+  IF (SameString( ARRAY(4), OldName )) THEN
+    IDXStart = 1
+    IDXEnd = 0
+    ARRAY(4)=NewName
+
+    ! NOTE: "ICOUNT - 1" b/c of deleting the 'Tdb' EMS function argument
+    DO J = 1, ICOUNT - 1, 1
+      LenToken = LEN_TRIM(ADJUSTL(ARRAY(J)))
+      IDXEnd = LenToken + IDXStart - 1
+      NewInOutArg(IDXStart:IDXEnd) = ARRAY(J)
```

Comment:
Yeah, this may need to be `TRIM(ADJUSTL(ARRAY(J)))`?.  I'm not sure.  I know different platforms might initialize data differently, but I'm surprised this specific thing would work on a platform and not the other.